.github/workflows/percy.yml

```
@@ -1,99 +1,21 @@
-        ports: ["6379:6379"]
-        options: --entrypoint redis-server
+    runs-on: ubuntu-latest
 
     env:
       RAILS_ENV: test
       DATABASE_URL: "postgres://rails:password@localhost:5432/rails_test"
 
     steps:
-     
       - name: Check out base repo code
         uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)